docs/windows.md

```diff
@@ -0,0 +1,158 @@
+    ```
+
+    **Alternatively**, for combined steps 3 and 4, you can use the following script. Remember to specify the correct version:
+
+    ```powershell
+    $version = "v0.13.0-rc2" # specify the release version, v0.13+
+    $arch = "amd64" # arm64 binary available too
+    curl.exe -LO https://github.com/moby/buildkit/releases/download/$version/buildkit-$version.windows-$arch.tar.gz
+    mv bin bin2
+    tar.exe xvf .\buildkit-$version.windows-$arch.tar.gz
+    # move them to an appropriate path in your $Env:PATH directories or:
+    Copy-Item -Path ".\bin" -Destination "$Env:ProgramFiles\buildkit" -Recurse -Force
+    # add `buildkitd.exe` and `buildctl.exe` binaries in the $Env:PATH
+    $Path = [Environment]::GetEnvironmentVariable("PATH", "Machine") + `
+        [IO.Path]::PathSeparator + "$Env:ProgramFiles\buildkit"
+    [Environment]::SetEnvironmentVariable( "Path", $Path, "Machine")
+    $Env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + `
+        [System.Environment]::GetEnvironmentVariable("Path","User")
+    ```
+1. Start `buildkitd.exe`, you should see something similar to:
+
+    ```
+    PS C:\> buildkitd.exe
+    time="2024-02-26T10:42:16+03:00" level=warning msg="using null network as the default"
+    time="2024-02-26T10:42:16+03:00" level=info msg="found worker \"zcy8j5dyjn3gztjv6gv9kn037\", labels=map[org.mobyproject.buildkit.worker.containerd.namespace:buildkit org.mobyproject.buildkit.worker.containerd.uuid:c30661c1-5115-45de-9277-a6386185a283 org.mobyproject.buildkit.worker.executor:containerd org.mobyproject.buildkit.worker.hostname:[deducted] org.mobyproject.buildkit.worker.network: org.mobyproject.buildkit.worker.selinux.enabled:false org.mobyproject.buildkit.worker.snapshotter:windows], platforms=[windows/amd64]"
+    time="2024-02-26T10:42:16+03:00" level=info msg="found 1 workers, default=\"zcy8j5dyjn3gztjv6gv9kn037\""
+    time="2024-02-26T10:42:16+03:00" level=warning msg="currently, only the default worker can be used."
+    time="2024-02-26T10:42:16+03:00" level=info msg="running server on //./pipe/buildkitd"
+    ```
+1. In another terminal (still elevated), try out a `buildctl` command to test that the setup is good:
+    ```
+    PS> buildctl debug info
+    BuildKit: github.com/moby/buildkit v0.0.0+unknown
+    ```
+    > **NOTE:** the version is `v0.0.0+unknown` since this is still a _release candidate (RC)_.
+
+### Example Build
+
+Now that everything is setup, let's build a [simple _hello world_ image](https://github.com/docker-library/hello-world/blob/master/amd64/hello-world/nanoserver-ltsc2022/Dockerfile).
+
+1. Create a directory called `sample_dockerfile`:
+    ```powershell
+    mkdir sample_dockerfile
+    cd sample_dockerfile
+    ```
+
+1. Inside it, add files `dockerfile` and `hello.txt`:
+    ```powershell
+    Set-Content Dockerfile @"
+    FROM mcr.microsoft.com/windows/nanoserver:ltsc2022
+    COPY hello.txt C:/
```

Comment:
Can we do a `RUN` command here? Otherwise this Dockerfile would build in linux as well.

Reply:
Done, ptal.